physionet-django/project/templates/project/published_project.html

```diff
@@ -87,7 +87,7 @@ <h1 class="form-signin-heading">{{ project.title }}</h1>
             <br><span>{{ citations.APA }}</span>
           </p>
           <div class="modal fade" id="citationModal" tabindex="-1" role="dialog" aria-labelledby="citationModalLabel" aria-hidden="true">
-            <div class="modal-dialog" role="document">
+            <div class="modal-dialog" role="document" style="max-width:800px">
```

Comment:
I think you'd planned on moving this style to the CSS?

Reply:
Hmmmm, I'm confused, I thought I fixed that with the last commit. Not sure where this branch is at now since it glitched out during the time github was down. I'll take a look at it.

Reply:
Ahhhh, maybe I forgot to change the project preview as well, I didn't notice both of them the first look at the changes!